var/spack/repos/builtin/packages/py-pacifica-namespace/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPacificaNamespace(PythonPackage):
+    """Python Pacifica Namespace Library"""
+
+    homepage = "https://github.com/pacifica/pacifica-namespace/"
+
+    version('0.0.2', sha256='a8f59aea1203a1557f7f57887b61e92f8450c74a8522798c5ddecf8fffb0b224', url="https://pypi.io/packages/source/p/pacifica-namespace/pacifica-namespace-0.0.2.tar.gz")
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
```

Comment:
Dep not needed

Reply:
i removed it on this one, and it works without it.